README.md

```diff
@@ -53,14 +53,23 @@ Add the following to your `tmux.conf` file to configure the tmux side of
 this customization.
 
 ``` tmux
-# Smart pane switching with awareness of vim splits
+# Smart pane switching with awareness of Vim splits.
 # See: https://github.com/christoomey/vim-tmux-navigator
-is_vim='echo "#{pane_current_command}" | grep -iqE "(^|\/)g?(view|n?vim?x?)(diff)?$"'
-bind -n C-h if-shell "$is_vim" "send-keys C-h" "select-pane -L"
-bind -n C-j if-shell "$is_vim" "send-keys C-j" "select-pane -D"
-bind -n C-k if-shell "$is_vim" "send-keys C-k" "select-pane -U"
-bind -n C-l if-shell "$is_vim" "send-keys C-l" "select-pane -R"
-bind -n C-\ if-shell "$is_vim" "send-keys C-\\" "select-pane -l"
+
+has_vim="ps -o state= -o comm= -t '#{pane_tty}' \
+    | grep -iEq '^[^TXZ ]+ +(\S+\/)?g?(view|n?vim?x?)(diff)?$'"
+tmux bind-key -n C-h  if-shell "$has_vim" "send-keys C-h"  "select-pane -L"
+tmux bind-key -n C-j  if-shell "$has_vim" "send-keys C-j"  "select-pane -D"
+tmux bind-key -n C-k  if-shell "$has_vim" "send-keys C-k"  "select-pane -U"
+tmux bind-key -n C-l  if-shell "$has_vim" "send-keys C-l"  "select-pane -R"
+tmux bind-key -n C-\\ if-shell "$has_vim" "send-keys C-\\" "select-pane -l"
+```
+
+The above code snippet is also available in the `vim-tmx-navigator.tmux` file,
+which you could alternatively include like this in your tmux config:
+
+```tmux
+run-shell /path/to/vim-tmux-navigator/vim-tmux-navigator.tmux
```

Comment:
I'd prefer to remove this and the associated file entirely from this PR, in favor of bringing those changes in later via the TPM configuration in works in #123.